Removed code that sets automatic backups to true during login

kg8z opened the pull request:
Issue: #2142 
<!-- Link to relevant issue. All PRs should be associated with an issue -->

## PR Type
What kind of change does this PR introduce?
<!-- Please uncomment one re more that apply to this PR -->

Bugfix
<!-- - Feature -->
<!-- - Code style update (formatting) -->
<!-- - Refactoring (no functional changes, no api changes) -->
<!-- - Build or CI related changes -->
<!-- - Documentation content changes -->
<!-- - Other... Please describe: -->


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
Automatic backups are enabled as part of login routine

## What is the new behavior?
Automatic backups are not enabled as part of login routine


## PR Checklist

Please check if your PR fulfills the following requirements:

<!-- If the code was not tested on all plattforms, please describe why. -->

- [x] Tested code on Windows
- [x] Tested code on Android
- [ ] Tested code on iOS
- [ ] Tests for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Contains **NO** breaking changes

Did not test on iOS but changes were in underlying code and should behave properly on all platforms.

<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

NPadrutt commented:
I think here is a test left that has to be adjusted to not check on that setting being set.

kg8z commented:
Well **that's** embarrassing. Total rookie mistake of not running the test scripts before submitting a PR. Sorry about that. Updated the failing test and added to the PR.

NPadrutt commented:
No worries. happens to the best. For that we have the automatic validation :)
